packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -21,7 +21,8 @@ const GlobalClassNames = {
 };
 
 export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
-  const { className, titleLabelClassName, theme, vertical, disabled, showTransitions, showValue } = props;
+  const { className, titleLabelClassName, theme, vertical, disabled, showTransitions, showValue, ranged } = props;
+  console.log('!!!!', props);
```

Comment:
Temporary console.log statement for checking why "ranged" props is not showing up. Will remove this once the problem is resolved